test/test_core.py

```diff
@@ -5986,8 +5986,18 @@ def test_fs_writeFile_wasmfs(self):
     self.emcc_args += ['-sFORCE_FILESYSTEM']
     self.do_run_in_out_file_test('fs/test_writeFile.cpp')
 
+  def test_fs_open_wasmfs(self):
+    self.emcc_args += ['-sWASMFS']
+    self.emcc_args += ['-sFORCE_FILESYSTEM']
+    self.do_runf(test_file('fs/test_open_wasmfs.c'), 'success')
+
   def test_fs_write(self):
     self.do_run_in_out_file_test('fs/test_write.cpp')
+  
+  def test_fs_write_wasmfs(self):
+    self.emcc_args += ['-sWASMFS']
+    self.emcc_args += ['-sFORCE_FILESYSTEM']
+    self.do_run_in_out_file_test('fs/test_write.cpp')
```

Comment:
There is an `@also_with_wasmfs_js` decorator you can attach to the previous test function instead of having to write a new test function here.